scripts/irods/test/test_delay_queue.py

```diff
@@ -432,8 +434,8 @@ def test_sigpipe_in_delay_server(self):
                 with open(server_config_filename, 'w') as f:
                     f.write(new_server_config)
 
-                # Bounce server to apply setting
-                irodsctl.restart(test_mode=True)
+                # Tell the server to apply setting
```

Comment:
This comment is unnecessary now. Your `reload_configuration()` function is clear.